homework.md

```diff
@@ -69,45 +74,47 @@ It send it to a friend
 What is a Pull Request?
 
 When you send a file over the internet
-
+What does a pull request means?
+A pull request  is an event that takes place in software development when a contributor/developer is ready to begin the process of merging new code changes with the main project repository.
 ## Question 12
 
 What is the Git History log?
 
 It lets you see into the future
```

Comment:
The Git History log is the history of all commits done on the repository , so that you can access these information from the header of the repo. the information avaialble in the log is : 
- Commit unique Id
- Commit Author or User
- Date and time of the commit
- Commit message